2/homework/requirements.txt

```diff
@@ -0,0 +1,146 @@
+# This file may be used to create an environment using:
```

Comment:
It is not commonly accepted to push your whole `pip freeze` or `conda list` in requirements file.
I would suggest to just add the packages you have in your imports, since pip handles dependencies, it would install necessary packages as well.
If you want to publish something like this I would rather suggest publishing your python environment,  or as an alternative build on top of miniconda